cmd/virt-controller/Dockerfile

```diff
@@ -18,6 +18,10 @@
 
 FROM centos:7
 
-COPY virt-controller /virt-controller
+# Create non-root user
+RUN useradd --create-home -s /bin/bash virt-controller
+WORKDIR /home/virt-controller
+USER virt-controller
+COPY virt-controller virt-controller
 
-ENTRYPOINT [ "/virt-controller" ]
+ENTRYPOINT [ "virt-controller" ]
```

Comment:
I'd prefer it if virt-controller were left at the root of the filesystem. It is referred to in other files. (e.g. manifests/virt-controller.in). Moving it to other locations is going to make it harder to use this docker image in general.

Currently this change causes the example deployment to fail with:
```
rpc error: code = 2 desc = failed to start container "45583db17aca696956bc10957efa114e69b2b2ffc240cf512e55dfb2c3eced58": Error response from daemon: {"message":"invalid header field value \"oci runtime error: container_linux.go:247: starting container process caused \\\"exec: \\\\\\\"/virt-controller\\\\\\\": stat /virt-controller: no such file or directory\\\"\\n\""}
```